pom.xml

```diff
@@ -571,6 +572,82 @@
             </tags>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>${maven.rewrite.plugin.version}</version>
+          <configuration>
+            <activeRecipes>
+              <recipe>org.checkstyle.openrewrite.DefinitionOfDone</recipe>
```

Comment:
thats inspired by @vlsi. Like approached in:

- https://github.com/junit-team/junit-framework/pull/4708#issuecomment-3056661395

You might like the automation by check itself, using rewrite.

The project might be something for you to use as automation in the future.

- https://github.com/checkstyle/checkstyle/pull/17490#issuecomment-3153188416


of course its open for suggestions as always.